packages/platform-sdk/source/services/transaction.service.ts

```diff
@@ -86,4 +87,8 @@ export class AbstractTransactionService implements Contract {
 	public async estimateExpiration(value?: string): Promise<string | undefined> {
 		return undefined;
 	}
+
+	protected toSatoshi = (value: NumberLike) => {
+		return BigNumber.make(value).toSatoshi(this.configRepository.get<number>("network.currency.decimals"));
```

Comment:
Inject the `BigNumberService` instead of constructing the same behaviour manually.